waftools/podselect.py

```diff
@@ -31,7 +31,10 @@ def perldoc_file(self):
 		self.bld.install_files(os.path.join(base_path, os.path.dirname(path)), outnode)
 
 class perldoc(Task.Task):
-	run_str='podselect ${SRC} > ${TGT}'
+	run_str='${PODSELECT} ${SRC} > ${TGT}'
 	color='BLUE'
 	ext_in=['.xs']
 	ext_out=['.pod']
+
+def configure(ctx):
+    ctx.find_program('podselect', var='PODSELECT')
```

Comment:
The change looks good. One tiny nit: Looks like this uses spaces for indentation instead of tabs. Is it easy to fix it for you? Otherwise I can do it after the merge.

Reply:
Good catch, fixed.